ldk/javascript/examples/react-loop/.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": ["airbnb-base", "plugin:prettier/recommended"],
```

Comment:
You might need to add
```json
  "parser": "babel-eslint",
  "parserOptions": {
    "ecmaFeatures": {
      "jsx": true
    }
  },
```

Does this Loop build and run fine without this?

Reply:
Added parser options ;9